cellcounter/cc_kapi/defaults.py

```diff
@@ -2,12 +2,36 @@
 
 from pytz import utc
 
-DEFAULT_KEYBOARD_MAP = {
+from .models import Keyboard
+
+BUILTIN_DESKTOP_KEYBOARD_MAP = {
+                        "label": u"Default desktop",
```

Comment:
Fixed in https://github.com/cellcounter/cellcounter/pull/446/commits/2125bfd1c0e479fb73afab0991173f25c42ce3f5